src/cdk-experimental/scrolling/virtual-scroll-viewport.ts

```diff
@@ -245,7 +246,28 @@ export class CdkVirtualScrollViewport implements OnInit, OnDestroy {
     }
   }
 
-  /** Sets the scroll offset on the viewport. */
+  /**  Scrolls to the offset on the viewport. */
+  scrollToOffset(offset: number, behavior: ScrollBehavior = 'auto') {
+    const viewportElement = this.elementRef.nativeElement;
+    const offsetDirection = this.orientation === 'horizontal' ? 'left' : 'top';
```

Comment:
Seems like this variable is only used when the scroll behavior is supported. Consider moving it into the `if`.